### PR TITLE
Changes needed for use in Mender

### PR DESCRIPTION
**Pull request by olehermanse**
Needed for Mender.

### CODE REVIEW

**Review by cf-bottom — CHANGES_REQUESTED** [review]
Please use a company e-mail instead of o***@gmail.com

**Comment by olehermanse** [comment]
@cf-bottom jenkins, please.

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5790/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5790/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5790/

**Comment by olehermanse** [comment]
Will be tested here: https://github.com/cfengine/core/pull/4352

File: configure.ac [hunk 1]
```diff
@@ -1482,14 +1482,12 @@ AC_MSG_RESULT( )
 AC_MSG_RESULT(Summary:)
 AC_MSG_RESULT(> Version: AC_PACKAGE_VERSION)
 
-AC_MSG_RESULT([> Required libraries])
+AC_MSG_RESULT([> Optional libraries])
 
 AC_MSG_RESULT([-> OpenSSL: $OPENSSL_PATH])
```
Comment (vpodzime) [inline configure.ac:1483]: Isn't OpenSSL required?

File: configure.ac [hunk 2]
```diff
@@ -1403,10 +1403,6 @@ if test "x$use_coverage" = "xyes"; then
     AC_MSG_ERROR([GCC is required for --enable-coverage])
   fi
 
-  if test "$debug" != "yes"; then
-    AC_MSG_ERROR([--enable-debug is required for --enable-coverage])
-  fi
```
Comment (vpodzime) [inline configure.ac:1461]: Does coverage work without debug?

**Review by vpodzime — COMMENTED** [review]
Looks good to me otherwise.

File: configure.ac (hunk 1 above)
Comment (olehermanse) [reply configure.ac:1483]: Nope, optional in pretty much the same way as PCRE.

File: configure.ac (hunk 2 above)
Comment (olehermanse) [reply configure.ac:1461]: I don't see any reason why it shouldn't.

File: configure.ac (hunk 1 above)
Comment (olehermanse) [reply configure.ac:1483]: https://github.com/cfengine/libntech/pull/63

Comment (olehermanse) [reply configure.ac:1483]: https://github.com/mendersoftware/mender-binary-delta/pull/43/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R48

File: configure.ac (hunk 2 above)
Comment (vpodzime) [reply configure.ac:1461]: I see many reasons, but we don't need/use it so :+1:

**Review by vpodzime — APPROVED** [review]
ACK

**Pull request merged**